### PR TITLE
Update screenshots for fonts to match current UI

### PR DESCRIPTION
**Pull request by emcguirk**
The current build of Godot uses a "Theme Overrides" instead of a series of "Custom..." Menus. This would align the 2D tutorial to match the current UI.

### CODE REVIEW

File: getting_started/first_2d_game/06.heads_up_display.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ well. There is a font file included in the game assets called
 Once you've done this on the ``ScoreLabel``, you can click the down arrow next
 to the Font property and choose "Copy", then "Paste" it in the same place
 on the other two Control nodes.
-Set "Custom Font Size" property of the ``ScoreLabel``. A setting of ``64`` works well.
+Set "Font Size" property of the ``ScoreLabel``. A setting of ``64`` works well.
```
Comment (pycbouh) [inline getting_started/first_2d_game/06.heads_up_display.rst:51]: Shouldn't it also refer to the "Theme Overrides" section?

```suggestion
Set the "Font Size" property of the ``ScoreLabel`` under "Theme Overrides > Font Sizes". A setting of ``64`` works well.
```

Comment (emcguirk) [reply getting_started/first_2d_game/06.heads_up_display.rst:51]: I had gone back and forth on whether to style it that way but I agree with the suggestion. It makes it more explicit and clear.

**Comment by mhilbrunner** [comment]
Thanks for contributing! Seems we got two PRs for the same change, this is now merged in https://github.com/godotengine/godot-docs/pull/5564

File: getting_started/first_2d_game/06.heads_up_display.rst [hunk 2]
```diff
@@ -48,7 +48,7 @@ well. There is a font file included in the game assets called
 Once you've done this on the ``ScoreLabel``, you can click the down arrow next
 to the Font property and choose "Copy", then "Paste" it in the same place
 on the other two Control nodes.
-Set "Custom Font Size" property of the ``ScoreLabel``. A setting of ``64`` works well.
+Set the "Font Size" property of the ``ScoreLabel`` under "Theme Overrides > Font Sizes". A setting of ``64`` works well.
```
Comment (pycbouh) [inline getting_started/first_2d_game/06.heads_up_display.rst:51]:
```suggestion
Set the "Font Size" property of the ``ScoreLabel`` under "Theme Overrides > Font Sizes".
A setting of ``64`` works well.
```

**Review by pycbouh — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks for contributing! 🎉

**Pull request merged**